<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ardoise</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="Licence" stdset="0">
   <string notr="true" extracomment="Licence">This file is part of Foobar.      Foobar is free software: you can redistribute it and/or modify     it under the terms of the GNU General Public License as published by     the Free Software Foundation, either version 3 of the License, or     (at your option) any later version.      Foobar is distributed in the hope that it will be useful,     but WITHOUT ANY WARRANTY; without even the implied warranty of     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the     GNU General Public License for more details.      You should have received a copy of the GNU General Public License     along with Foobar.  If not, see &lt;http://www.gnu.org/licenses/&gt;.</string>
  </property>
  <widget class="Ardoise" name="ardoise"/>
  <widget class="QDockWidget" name="doc">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>240</red>
         <green>239</green>
         <blue>239</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="74">
         <red>108</red>
         <green>107</green>
         <blue>107</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="129">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <widget class="QPushButton" name="open">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>41</y>
       <width>106</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Ouvrir</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/open</normaloff>:/open</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string notr="true">Ctrl+O</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="statut">
     <property name="geometry">
      <rect>
       <x>583</x>
       <y>41</y>
       <width>16</width>
       <height>19</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="erase">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>136</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>RÃ©initialiser</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/erase</normaloff>:/erase</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string notr="true">Del</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="dispSel">
     <property name="geometry">
      <rect>
       <x>251</x>
       <y>41</y>
       <width>188</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Afficher la selection</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/sel</normaloff>:/sel</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>41</y>
       <width>135</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>135</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Enregistrer</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/save</normaloff>:/save</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string notr="true">Ctrl+S</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="w1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>5</y>
       <width>93</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="suffix">
      <string notr="true"> px</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="value">
      <double>5.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="couleur1">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>134</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>134</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Couleur 1</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/pal</normaloff>:/pal</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string notr="true">1</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="w2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>5</y>
       <width>93</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="suffix">
      <string notr="true"> px</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="value">
      <double>20.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="couleur2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>5</y>
       <width>134</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>134</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Sans Serif</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Couleur 2</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/pal</normaloff>:/pal</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string notr="true">2</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="swapMode_pb">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>10</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string notr="true">SwapMode</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <zorder>doc</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Ardoise</class>
   <extends>QWidget</extends>
   <header>ardoise.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>couleur1</sender>
   <signal>clicked(bool)</signal>
   <receiver>mainWindow</receiver>
   <slot>setCol1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>w1</sender>
   <signal>valueChanged(double)</signal>
   <receiver>mainWindow</receiver>
   <slot>setWeight1(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>couleur2</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>setCol2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>w2</sender>
   <signal>valueChanged(double)</signal>
   <receiver>mainWindow</receiver>
   <slot>setWeight2(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setCol1()</slot>
  <slot>setWeight1(double)</slot>
  <slot>setCol2()</slot>
  <slot>setWeight2(double)</slot>
  <slot>swapMode()</slot>
 </slots>
</ui>
